imports/plugins/core/versions/server/migrations/7_add_shop_slugs_to_schema.js

```diff
@@ -15,7 +15,7 @@ Migrations.add({
     // Loop through all shops creating slugs and creating merchant shop objects
     shops.forEach((shop) => {
       // create slug from shop name
-      const shopSlug = getSlug(shop.name);
+      const shopSlug = slugify(shop.name);
```

Comment:
This is the fix. Everything else is extra prevention for migration errors that I saw while testing, but that aren't very likely in the real world.